README.md

```diff
@@ -182,6 +182,14 @@ Deploy to Staging is manual. Training of new users is done in staging.
 
 https://casa-r4g-staging.herokuapp.com/
 
+### QA environment
+
+QA is done on the staging servers
+
+- https://casa-r4g-staging.herokuapp.com/
```

Comment:
That is our staging server. QA is actually https://casa-qa.herokuapp.com/

Reply:
@jmkoni thanks i'll update that. Do the same credentials apply to QA server?

Reply:
Yes

Reply:
🙃 this should be good to go then